packages/botonic-react/src/utils.js

```diff
@@ -126,3 +126,14 @@ export const getWebchatElement = () =>
 export const setWebchatElementHeight = newHeight => {
   getWebchatElement().style.height = newHeight
 }
+
+export const serializeRegexs = (_, value) => {
+  if (value instanceof RegExp) return value.toString()
+  return value
+}
+
+export const rehydrateRegex = regex => {
+  const fragments = regex.match(/\/(.*?)\/([a-z]*)?$/i)
+  const rehydrated = new RegExp(fragments[1], fragments[2] || '')
+  return rehydrated
+}
```

Comment:
Good one! I like the simplicity and the naming. Since we can't use typescript here, I'd suggest... 
- `regex` -> `regexStr`
- You can improve the regex with this tiny change `/\/(.[^\/]+)\/([a-z]{1})?$/i`
   - At least 1 character on the first part
   - Exclude the `/` to properly have the first part grouped
   - The second part only needs `{1}` character

Test it, it should work!

Reply:
it seems to work good! i have applied both changes, thanks!